Tests/IndexerTest.php

```diff
@@ -401,4 +404,210 @@ public function ifUrlIsNotIndexedReturnTrue()
 
         $this->assertTrue($actual);
     }
+
+    public function testGetHashSolarId()
+    {
+        $url = 'https://github.com';
+        $hashIdUrl = sha1(strtolower(UrlCheck::fixUrl($url)));
+
+        $uriObj = new Uri('https://github.com');
+        $hashIdUri = sha1(strtolower(UrlCheck::fixUrl($uriObj->toString())));
+
+        $solrClient = $this->getMockBuilder('Solarium\Client')
+            ->setConstructorArgs([])
+            ->setMethods(['createSelect', 'select'])
+            ->getMock();
+
+        $indexer = new Indexer($solrClient, [], 50);
+
+        $this->assertEquals($indexer->getHashSolarId($url), $hashIdUrl);
+        $this->assertEquals($indexer->getHashSolarId($uriObj), $hashIdUri);
+        $this->assertEquals($hashIdUrl, $hashIdUri);
+    }
+
+    public function storedAndNotExpiredDataProvider()
+    {
+        $url1 = 'http://www.simgroep.nl/internet/medewerkers_41499/';
+        $url2 = 'http://www.simgroep.nl/internet/nieuws-uit-de-branche_41509/';
+        $url3 = 'http://www.simgroep.nl/internet/portfolio_41515/search/';
+        $url4 = 'http://www.simgroep.nl/internet/vacatures_41521/';
```

Comment:
could one of them dont have a / at end to see if UrlCheck working properly and will not be a problem with slashes at end